cmd/benchmark/main.go

```diff
@@ -253,11 +265,11 @@ func getTotalBalance(insSDK *sdk.SDK, members []sdk.Member) (totalBalance *big.I
 		totalBalance.Add(b, res.balance)
 	}
 
-	return totalBalance
+	return totalBalance, membersWithBalanceMap
 }
 
 func getMembers(insSDK *sdk.SDK, number int, migration bool) ([]sdk.Member, error) {
-	var members = make([]sdk.Member, number)
+	var members = make([]sdk.Member, number+1)
```

Comment:
Where did +1 came from?

Reply:
We load from file not just number of members, but also migration admin or fee member. Add comment and rewrite it for loadMembers branch only.